embassy-stm32/src/usart/mod.rs

```diff
@@ -303,6 +612,15 @@ impl<'d, T: BasicInstance, TxDma, RxDma> Uart<'d, T, TxDma, RxDma> {
     }
 }
 
+fn configure(r: &Regs, config: &Config, pclk_freq: Hertz, multiplier: u32) {
```

Comment:
`Regs` is `Copy`, you should pass it by value. (it is a pointer internally already)

Reply:
Sorry about that. fixed

Reply:
No need to say sorry! :)